pandasaurus_cxg/schema/cell_x_gene_schema.py

```diff
@@ -0,0 +1,23 @@
+required_fields = [
```

Comment:
Looking at files on CxG, this might be too strict.  Happy to go with this for now I think.

Reply:
We should investigate integrating their validator.  lib is on PyPi.  For now probably sufficient to just specify the fields we need for functionality.